stages/9-openebs-install-check/openebs-self-installation-check

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -ex
+
+######################
+##   Prerequisites  ##
+######################
+
+path=$(pwd)
+# Copy kubeconfig
+mkdir ~/.kube
+cp  $path/.gcp/config ~/.kube/config
```

Comment:
To use the kube config of DOP installed k8s cluster, copy the following config file -- `$path/.kube/config`.